src/redux/books/booksSlice.js

```diff
@@ -1,37 +1,78 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
 
-const initialState = [
-  {
-    item_id: 'item1',
-    title: 'The Great Gatsby',
-    author: 'John Smith',
-    category: 'Fiction',
-  },
-  {
-    item_id: 'item2',
-    title: 'Anna Karenina',
-    author: 'Leo Tolstoy',
-    category: 'Fiction',
-  },
-  {
-    item_id: 'item3',
-    title: 'The Selfish Gene',
-    author: 'Richard Dawkins',
-    category: 'Nonfiction',
+const initialState = {
+  books: [],
+  status: 'idle',
+  error: null,
+};
+
+export const fetchBooks = createAsyncThunk('books/fetchBooks', async () => {
+  const response = await axios.get('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/HGGa9fmUyjETuK6YcEa3/books');
+  const data = Object.entries(response.data).map(([id, book]) => ({ id, ...book[0] }));
+  return data;
+});
+
+export const createBook = createAsyncThunk('books/createBook', async (bookData) => {
+  const response = await axios.post(
+    'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/HGGa9fmUyjETuK6YcEa3/books',
+    {
+      item_id: bookData.id,
+      ...bookData,
+    },
+    {
+      headers: { 'Content-Type': 'application/json' },
+    },
+  );
+  return { ...bookData, id: response.data.item_id };
+});
+
+/* eslint-disable */
```

Comment:
- [ ] As stated before, avoid disabling linters

Reply:
fixed! I forgot to erase those!